Dockerfile

```diff
@@ -28,16 +29,21 @@ SHELL ["/bin/bash", "-c"]
 # Install commands used in our scripts and standard present on a clean ubuntu
 # installation and setup a user with sudo priviledges
 RUN apt-get update -qq && \
-    apt-get install -qq --assume-yes --no-install-recommends apt-transport-https apt-utils ca-certificates curl dbus dialog git lsb-release sudo tzdata wget && \
+    apt-get install -qq --assume-yes --no-install-recommends apt-transport-https apt-utils ca-certificates curl dbus dialog git lsb-release openssh-client sudo tzdata wget > /dev/null && \
     # Add amigo user
-    adduser --disabled-password --gecos "" $USER && \
-    adduser $USER sudo && \
-    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+    adduser -u 1000 --disabled-password --gecos "" $USER && \
+    usermod -aG sudo $USER && \
+    usermod -aG adm $USER && \
+    echo "%sudo ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/"$USER"
 
 # Setup the current user and its home directory
 USER "$USER"
 WORKDIR /home/"$USER"
 
+ADD installer/bootstrap.bash ./bootstrap.bash
+
+RUN mkdir -p -m 0700 ~/.ssh && ssh-keyscan gitlab.com >> ~/.ssh/known_hosts && chmod 644 ~/.ssh/known_hosts
```

Comment:
This should inherit from travis known hosts file

Reply:
Are you sure? Because if it does then all the things on the host file of travis will get added into the docker image. With this line we restrict it the hosts to what we want.

Reply:
Are there may entries in the known hosts file? But people could overwrite the entire file to a limited set. But also we need to move it to the place where we use it. Which is not in tue-env

Reply:
Yeah there a many entries in a known hosts file. Also as they are generated by ssh commands, there is no easy way of extracting a part from it.

I don't understand why you say we need to move this to somewhere else. This is only in the docker image not a part of tue-env in general. Doing this properly would mean integrating this in tue-env by parsing git url and checking if the ssh host exists in known host file, if not then adding it, which in my opinion is not in the scope of this PR.

Reply:
This is what is in the Travis known_hosts file
https://travis-ci.com/github/tue-robotics/hero_bringup/jobs/297688538#L183

Reply:
Azure doesn’t have a known hosts file by default

Reply:
We could try mounting the file from the CI to docker container. That would resolve your issue with the explicit declaration of known hosts but I'll need to think how to simplify the build process. Mounting would work perfectly for `install-packages.sh` but in `build-docker-image.sh` that would be a problem. But we can skip it for now as we are not using ssh to build images.

Reply:
Script to merge known_hosts file: https://blog.backslasher.net/merge-known-hosts.html

Reply:
Merging is added